### ISSUE TITLE
Some side-effect imports are not autofixable

### ISSUE THREAD

**Issue by izhan** [lydell/eslint-plugin-simple-import-sort#34]
Hi there - first off, thank you for creating this project!

We are having autofix issues when running `eslint --fix` on this particular set of side-effect imports:
```js
import "codemirror/addon/fold/brace-fold"
import "codemirror/addon/edit/closebrackets"
import "codemirror/addon/fold/foldgutter"
import "codemirror/addon/fold/foldgutter.css"
import "codemirror/addon/lint/json-lint"
import "codemirror/addon/lint/lint"
import "codemirror/addon/lint/lint.css"
import "codemirror/addon/scroll/simplescrollbars"
import "codemirror/addon/scroll/simplescrollbars.css"
import "codemirror/lib/codemirror.css"
import "codemirror/mode/javascript/javascript"
```

**Expected behavior**:
- no errors are emitted
- side-effect imports are not reordered

**Actual behavior**:
- error (Run autofix to sort...)
- side-effect imports are reordered like below

```js
import "codemirror/addon/lint/lint"
import "codemirror/addon/fold/brace-fold"
import "codemirror/addon/fold/foldgutter"
import "codemirror/addon/fold/foldgutter.css"
import "codemirror/addon/lint/json-lint"
import "codemirror/addon/edit/closebrackets"
import "codemirror/addon/lint/lint.css"
import "codemirror/addon/scroll/simplescrollbars"
import "codemirror/addon/scroll/simplescrollbars.css"
import "codemirror/lib/codemirror.css"
import "codemirror/mode/javascript/javascript"
```

If I find an easier repro, I'll include it. Happy to also help dig into this further when I have a chance. Cheers.

**Comment by lydell** [comment]
Hi! I pasted those imports quickly into a project that uses eslint-plugin-simple-import-sort and their order was preserved. What am I missing? There must be something different in your environment. Could you please make a reproduction repo? Try to remove stuff until it doesn’t happen anymore etc.

**Comment by izhan** [comment]
Weird, let me continue to dig into further...thanks for the quick reply!

**Comment by izhan** [comment]
Here's a repro: https://github.com/izhan/eslint-plugin-simple-import-sort-repro. Running `npm run test` should produce an error for `failing-import.js`

Interestingly, deleting any line in `failing-import.js` will cause the autofixer to work again.

**Comment by lydell** [comment]
Thanks! `npm run test` produces no errors for me though:

```
~/stuff/eslint-plugin-simple-import-sort-repro master
❯ npm test

> eslint-plugin-simple-import-sort-repro@1.0.0 test /Users/lydell/stuff/eslint-plugin-simple-import-sort-repro
> eslint --fix ./failing-import.js


~/stuff/eslint-plugin-simple-import-sort-repro master
❯
```

**Comment by izhan** [comment]
Ah crap...
```
$ node --version
v10.16.3
```

Looks like stable sorting was released in Node v12+ (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort). Updating to v12.13.1 solved this issue for me.

Should we change the minimum Node version for this repo? https://github.com/lydell/eslint-plugin-simple-import-sort#development Or perhaps just implement a stable sort for `sort.js`?

**Comment by lydell** [comment]
Good catch! I think we should support Node 10. It might be as easy as changing `0` to `itemA.index - itemB.index` here: https://github.com/lydell/eslint-plugin-simple-import-sort/blob/15ad0313109930aa98bc070ed045a1418d2eb557/src/sort.js#L779

**Comment by lydell** [comment]
Released in v5.0.1.
